src/raftstore/store/worker/pd.rs

```diff
@@ -769,6 +776,25 @@ impl<T: PdClient> Runner<T> {
         self.store_stat.store_read_io_rates = read_io_rates;
         self.store_stat.store_write_io_rates = write_io_rates;
     }
+
+    fn handle_refresh_config(&mut self, handle: &Handle) {
+        let config_client = &mut self.config_client;
+        info!("refresh config"; "component id" => config_client.get_id(), "version" => ?config_client.get_version());
+        if let Err(e) = config_client.refresh_config(self.pd_client.clone()) {
+            error!("failed to refresh config"; "component id" => config_client.get_id(), "version" => ?config_client.get_version(), "err" => ?e)
+        }
+        let scheduler = self.scheduler.clone();
+        let when = Instant::now() + config_client.get_refresh_interval();
```

Comment:
Can it be changed dynamically?

Reply:
Yes it can, we call `refresh_config` first and then `get_refresh_interval`.